routes/cube_routes.js

```diff
@@ -1182,6 +1192,7 @@ router.post('/uploaddecklist/:id', ensureAuth, async (req, res) => {
     deck.comments = [];
     deck.cubename = cube.name;
     deck.cube = cube._id;
+    deck.cubeOWner = cube.owner;
```

Comment:
This and other instances in the file should be `cubeOwner` without the W capitalized.